classes/class-astra-blk-meta-boxes-bulk-edit.php

```diff
@@ -607,6 +611,11 @@ public function display_quick_edit_custom( $column, $screen ) {
 		 */
 		public function enqueue_admin_scripts_and_styles() {
 			wp_enqueue_style( 'astra-blk-admin', ASTRA_BLK_URI . 'assets/css/astra-admin.css', array(), ASTRA_BLK_VER );
+
+			if ( ! current_user_can( 'edit_posts' ) ) {
```

Comment:
@imnavanath Should this be the first line of function? Should we enqueue `astra-blk-admin` CSS only when user has permissions?

Reply:
If we do so @premanshup then other users (! current_user_can( 'edit_posts' )) will directly see this column and it will look very weird. So I don't think so we should move this condition before enqueue style.

Reply:
@imnavanath I mean do we need to put 
`wp_enqueue_style( 'astra-blk-admin', ASTRA_BLK_URI . 'assets/css/astra-admin.css', array(), ASTRA_BLK_VER );`
 this code under condition as well? Should we load this CSS only when user has permission?

Reply:
The CSS file is important. So we can keep this enqueue style.

Reply:
Okay!